src/WebJobs.Script/Rpc/FunctionRegistration/FunctionRegistry.cs

```diff
@@ -66,8 +66,8 @@ public void Register(FunctionRegistrationContext context)
 
         public void WorkerError(WorkerErrorEvent workerError)
         {
-            ILanguageWorkerChannel erroredChannel = _channelsDictionary[workerError.WorkerId];
-            if (erroredChannel != null)
+            ILanguageWorkerChannel erroredChannel;
+            if (_channelsDictionary.TryGetValue(workerError.WorkerId, out erroredChannel) && erroredChannel != null)
```

Comment:
I don't think you need the `erroredChannel != null` check now - if Try returns true it won't be null

Reply:
Correct. Also, you can inline this variable declaration (e.g. `out ILanguageWorkerChannel erroredChannel`)

Reply:
I'm seeing other null refs locally when running, indexing into _channelsDictionary by worker ID